lochness/email/__init__.py

```diff
@@ -156,6 +156,21 @@ def send_out_daily_updates(Lochness, days: int = 1, test: bool = False):
                                   'Protected', 'Study', 'Processed', 'Subject',
                                   'Datatype', 'Download time (UTC)']
         s3_df_selected.reset_index().drop('index', axis=1, inplace=True)
+
+        # too many dicom file names -> remove
+        subject_mri_gb = s3_df_selected[
+                s3_df_selected.Datatype == 'mri'].groupby('subject')
+
+        sample_mri_df = pd.DataFrame()
+        for _, table in subject_mri_gb:
+            subject_mri_sample = table.iloc[:2]  # select only two raws
```

Comment:
Is this the shorthand for `subject_mri_sample.iloc[0]`?

Reply:
no, `.iloc[0]` returns the first row in `series` format, `.iloc[:2]` returns the first two rows in `dataframe` format.

Reply:
Sorry, that's not what I asked. In the next line, you are assigning a value to `subject_mri_sample.iloc[1]` which is 2nd row. What row are you assigning by `subject_mri_sample` only?

Reply:
oh I'm trying to keep one dicom file name followed by '...'., for each subject. Does it answer your question?

Reply:
Tashrif is answering his question:

* `subject_mri_sample` data frame has only two rows. This variable was not declared before. It is declared and assigned in one step as a data frame from right hand side.
* `subject_mri_sample.iloc[1]['File name'] = '...'` overwrites the `File name` in 2nd row with ellipsis